pom.xml

```diff
@@ -101,6 +101,16 @@
             <artifactId>flink-metrics-core</artifactId>
             <version>1.13.0</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.flink</groupId>
+            <artifactId>flink-streaming-java_2.11</artifactId>
+            <version>1.13.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.flink</groupId>
+            <artifactId>flink-runtime_2.11</artifactId>
+            <version>1.13.0</version>
```

Comment:
nit. add a property "flink.version" and use it everywhere you have 1.13.0. See pubsublite-beam-io for an example.

Reply:
Will do in another pr, tianzi suggested that I do this for pubsublite too